client/components/addReview.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { postNewReview } from '../store';
+
+class AddReview extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      description: '',
+      rating: 5
+    };
+    this.changeHandler = this.changeHandler.bind(this);
+    this.submitHandler = this.submitHandler.bind(this);
+  }
+  changeHandler(evt) {
+    const newState = {};
+    newState[evt.target.name] = evt.target.value;
+    this.setState(newState);
+  }
+  submitHandler(evt) {
+    evt.preventDefault();
+    const { userId, productId } = this.props;
+    const review = { ...this.state, userId, rating: +this.state.rating };
+    this.props.submitDispatch(productId, review);
+    this.setState({title: '', description: '', rating: 5})
+  }
+  render() {
+    const isItValid = this.state.title && this.state.description;
```

Comment:
FYI: On the backend, only a description is required. With the review seed data, a lot of the reviews do not have titles.